lib/package.js

```diff
@@ -3,123 +3,73 @@
 var assign = require('object-assign');
 var debug = require('diagnostics')('warehouse-models:package');
 var difference = require('lodash.difference');
+const Joi = require('joi');
+const Dynastar = require('dynastar');
 
 /**
- * function package(datastar, models)
- * Returns an  datastar model representing
+ * function package(dynamo)
+ * Returns an  dynamo model representing
  * the metadata of a single version of a package
  * being persisted to the connection associated
- * with the `datastar` instance provided.
+ * with the `dynamo` instance provided.
  *
- * @param {datastar} datastar Instance of datastar to attach to.
- * @param {Object} models Set of registry-data models.
+ * @param {dynamo} dynamo Instance of dynamo to attach to.
  *
- * @returns {Object} an datastar model representing
+ * @returns {Object} an dynamo model representing
  * the metadata of a single version of a package
  */
-module.exports = function pkg(datastar, models) {
-  var cql = datastar.schema.cql;
-  var Pkg = datastar.define('package', {
-    schema: datastar.schema.object({
-      name: cql.text(),
-      version: cql.text(),
-      description: cql.text(),
-      main: cql.text(),
+module.exports = function pkg(dynamo) {
+  const hashKey = 'name';
+  const model = dynamo.define('Package', {
+    hashKey,
+    tableName: 'WarehousePackage',
+    schema: {
```

Comment:
define this as its own variable above. `const schema = { }` and pass it in as `schema` in this `dynamo.define`, we need to use `Object.keys(schema)` to replace the previous `model.schema.mappedFields` defined below